### PR TITLE
Small fixes to latex-related docs.

### PR DESCRIPTION
**Pull request by anntzer**
- Capitalization, spacing.
- Make bullet points consistently not end with period.

## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: tutorials/text/usetex.py [hunk 1]
```diff
@@ -105,7 +105,7 @@
   packages to get all the goodies that come bundled with other latex
   distributions.
 
-* Some progress has been made so matplotlib uses the dvi files
+* Some progress has been made so Matplotlib uses the dvi files
   directly for text layout. This allows latex to be used for text
   layout with the pdf and svg backends, as well as the \*Agg and PS
   backends. In the future, a latex installation may be the only
```
Comment (smartlixx) [inline tutorials/text/usetex.py:111]:
```suggestion
  backends. In the future, a LaTeX installation may be the only
```

Comment (anntzer) [reply tutorials/text/usetex.py]: sure

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
If I merge now, I'll have to rebase #20011 🙄

**Comment by anntzer** [comment]
no worries, I can rebase over yours.

**Comment by timhoffm** [comment]
Ah, whatever. It's probably easier for me as it's just copy & paste the whole section.

**Pull request merged**